### DISCUSSION TITLE
🐺 v3.6.x

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#5106] [Releases]
<div align="center">

| Manager | Buddy | Feature Branch | Project |
| :---: | :---: | :---: | :---: |
| @bjlittle | @ESadek-MO | [v3.6.x](https://github.com/SciTools/iris/tree/v3.6.x) | [![GH Project](https://img.shields.io/badge/project-%F0%9F%90%BA%20v3.6.x-yellow?logo=github&logoColor=lightgrey)](https://github.com/orgs/SciTools/projects/11) |

</div>

<div align="center">

| Minor Release | Status | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.6.0rc0` | [![status](https://img.shields.io/badge/v3.6.0rc0-released-green)](https://github.com/SciTools/iris/discussions/5106) | `02-05-2023` | [Docs](https://scitools-iris.readthedocs.io/en/v3.6.0rc0/index.html) | [v3.6.0rc0](https://github.com/SciTools/iris/releases/tag/v3.6.0rc0) | [rc_iris](https://anaconda.org/conda-forge/iris?version=3.6.0rc0&channel=rc_iris) | [scitools-iris (--pre)](https://pypi.org/project/scitools-iris/3.6.0rc0/) |
| `v3.6.0` | [![status](https://img.shields.io/badge/v3.6.0-released-green)](https://github.com/SciTools/iris/discussions/5106) | `17-05-2023` + `1 day` | [Docs](https://scitools-iris.readthedocs.io/en/v3.6.0/index.html) | [v3.6.0](https://github.com/SciTools/iris/releases/tag/v3.6.0) | [iris](https://anaconda.org/conda-forge/iris?version=3.6.0&channel=main) | [scitools-iris](https://pypi.org/project/scitools-iris/3.6.0/) |

| Patch Release | Status | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.6.1` | [![status](https://img.shields.io/badge/v3.6.1-released-green)](https://github.com/SciTools/iris/discussions/5106) | `26-06-2023` | [Docs](https://scitools-iris.readthedocs.io/en/v3.6.1/index.html) | [v3.6.1](https://github.com/SciTools/iris/releases/tag/v3.6.1) | [iris](https://anaconda.org/conda-forge/iris?version=3.6.1&channel=main) | [scitools-iris](https://pypi.org/project/scitools-iris/3.6.1/) |

</div>

**Comment by rcomer** [comment]
Any thoughts on dropping python 3.8 support for this release?  It already fell off the the NEP29 schedule.
https://numpy.org/neps/nep-0029-deprecation_policy.html

**Reply by bjlittle** [reply]
@rcomer Already done! Thanks to you :star_struck:

**Comment by bjlittle** [comment]
The `iris` minor release candidate **v3.6.0rc0** is now available. All feedback welcome :tada: 

We've scheduled a **2 week** window for the community to provide feedback before we tag the **v3.6.0** minor release on `17-05-2023`.

Enjoy :beers:

**Reply by bouweandela** [reply]
Thanks @bjlittle! Maybe the release notes could stress the added support for using the Dask [distributed](https://distributed.dask.org) scheduler a bit more. This will make it possible to run iris on Dask clusters, which should benefit anyone who has a large amount of data to crunch.

**Reply by bjlittle** [reply]
Good call :+1: 

Will do, thanks!

**Comment by bjlittle** [comment]
Apologies, but after an unexpected delay of 1 day, I'm starting to crank the handle on the `iris` 3.6.0 release...

**Reply by bjlittle** [reply]
`iris` v3.6.0 is now officially out there... come get some!

Enjoy :partying_face:

**Comment by bjlittle** [comment]
Thanks for being an awesome release buddy @ESadek-MO :heart:

**Comment by schlunma** [comment]
These are PRs which are relevant for a potential bugfix release from ESMValTool devs:

- https://github.com/SciTools/iris/pull/5340
- https://github.com/SciTools/iris/pull/5142
- https://github.com/SciTools/iris/pull/5341
- #5349

Thanks all, we really appreciate your support :heart:

**Reply by bjlittle** [reply]
@schlunma Do you need all of these PRs banked for your ESMValTool 2.9.0 release?

Those that you do need merged and available in an `iris` 3.6.1 release should all target the `v3.6.x` feature release branch instead of `main`.

I'm assuming that you can't wait until `iris` 3.7.0, which is scheduled for release in Aug'23, right?

**Reply by bjlittle** [reply]
@schlunma and @bouweandela 

Please use this GH Discussion to coordinate any PRs that you wish to be included in an `iris` 3.6.1 release :+1:

**Reply by schlunma** [reply]
> @schlunma Do you need all of these PRs banked for your ESMValTool 2.9.0 release?

Yes!
 
> Those that you do need merged and available in an `iris` 3.6.1 release should all target the `v3.6.x` feature release branch instead of `main`.

Just did that for https://github.com/SciTools/iris/pull/5340 and https://github.com/SciTools/iris/pull/5341. @sloosvel could you do the same for https://github.com/SciTools/iris/pull/5142 if you want that included in 3.6.1, please?

Thanks so much @bjlittle!!

**Comment by bjlittle** [comment]
@bouweandela @schlunma @valeriupredoi @sloosvel

`iris` v3.6.1 is now available, happy testing!

Hope your `esmvaltool` release is now unblocked :+1:

**Reply by schlunma** [reply]
Thank you so much for your continuous support, that's highly appreciated ❤️
